test/v3/addLiquidityBoosted/addLiquidityPartialBoosted.integration.test.ts

```diff
@@ -52,16 +54,20 @@ describe('V3 add liquidity partial boosted', () => {
     let rpcUrl: string;
     let client: PublicWalletClient & TestActions;
     let testAddress: Address;
+    let snapshot: Hex;
     const addLiquidityBoosted = new AddLiquidityBoostedV3();
-    let addLiquidityInput: AddLiquidityBoostedUnbalancedInput;
     const amountsIn = [
         TokenAmount.fromHumanAmount(usdtToken, '1'),
         TokenAmount.fromHumanAmount(daiToken, '2'),
     ];
 
     beforeAll(async () => {
         // setup chain and test client
-        ({ rpcUrl } = await startFork(ANVIL_NETWORKS.SEPOLIA));
+        ({ rpcUrl } = await startFork(
+            ANVIL_NETWORKS.SEPOLIA,
+            undefined,
+            7010800n,
```

Comment:
Is the block number fixed here to protect against external changes to pool liquidity that could cause test to fail?

Looks like the `addLiquidityBoosted.integration.test.ts` file does not specify a block number when starting the fork. Is there some reason that is okay there but not here? 

Or is it better practice to always fix the blockNumber the fork starts? Or is that not worth the trade off of having to update block numbers each time a new version of v3 is deployed? 

Maybe could make sense to centralize the block all v3 tests run at using a constants file?

Reply:
Yeah, I had to fix the blockNumber due to how tests were setup (see line 220).
On this addLiquidityPartialBoosted we're testing the queryOutput separated from the actual transaction, so we can't compare to token balance delta as we do on other files.
For add liquidity unbalanced that's ok, because we can compare queryOutputs with amountsIn provided as input, but for add liquidity proportional, we don't have that.
So, if we want to check queryOutput against all amountsIn, we have to provide a known/hardcoded value, which requires blockNumber to be fixed.
With that being said, I usually prefer to avoid fixing blockNumbers as well.
We could refactor addLiquidityPartialBoosted to follow the same pattern as addLiquidityBoosted and this would be solved.
What do you think?